yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/extension/LocationConstraintContent.java

```diff
@@ -0,0 +1,65 @@
+package com.yoti.api.client.shareurl.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LocationConstraintContent {
+
+    private final DeviceLocation expectedDeviceLocation;
+
+    LocationConstraintContent(double latitude, double longitude, double radius, double maxUncertainty) {
+        this.expectedDeviceLocation = new DeviceLocation(latitude, longitude, radius, maxUncertainty);
+    }
+
+    @JsonProperty("expected_device_location")
+    public DeviceLocation getExpectedDeviceLocation() {
+        return expectedDeviceLocation;
+    }
+
+    public static final class DeviceLocation {
+
+        private final double latitude;
+        private final double longitude;
+        private final double radius;
+        private final double maxUncertainty;
+
+        DeviceLocation(double latitude, double longitude, double radius, double maxUncertainty) {
+            this.latitude = latitude;
+            this.longitude = longitude;
+            this.radius = radius;
+            this.maxUncertainty = maxUncertainty;
+        }
+
+        /**
+         * Latitude of the user's expected location
+         */
+        @JsonProperty("latitude")
+        public double getLatitude() {
+            return latitude;
+        }
+
+        /**
+         * Longitude of the user's expected location
+         */
+        @JsonProperty("longitude")
+        public double getLongitude() {
+            return longitude;
+        }
+
+        /**
+         * Acceptable distance from the given lat/long position, in metres
+         */
+        @JsonProperty("radius")
+        public double getRadius() {
+            return radius;
+        }
+
+        /**
+         * Maximum acceptable margin of error, in metres
```

Comment:
This is the maximum acceptable margin of error in the device's location, right? I had to think about whether this was needed in addition to radius for a moment before I realised, so it might be better if we clarify that in the comments, even though I've seen there is more detail in LocationConstraintExtensionBuilder.java

Reply:
Ok, I've used the comments from the builder instead.